lib/components/secure-payment-component.js

```diff
@@ -150,16 +140,24 @@ export default class SecurePaymentComponent extends AsyncBaseContainer {
 	async toggleCartSummary() {
 		// Mobile
 		if ( currentScreenSize() === 'mobile' ) {
-			return await driverHelper.clickWhenClickable(
+			await driverHelper.clickWhenClickable( this.driver, By.css( '.checkout__summary-toggle' ) );
+
+			return driverHelper.waitTillPresentAndDisplayed(
 				this.driver,
 				By.css( '.checkout__summary-toggle' )
 			);
 		}
 	}
 
 	async cartTotalAmount() {
+		let cartElement = await this.driver.findElement( By.css( '.cart-total-amount' ) );
+
+		if ( currentScreenSize() === 'mobile' ) {
+			await this.driver.executeScript( 'arguments[0].scrollIntoView()', cartElement );
```

Comment:
Had some trouble with the cart not coming into view on mobile causing other steps to fail. Explicitly bringing into view resolved that for me

Reply:
Does [`scrollIntoView()`](https://github.com/Automattic/wp-e2e-tests/blob/master/lib/driver-helper.js#L414) from _driver-helper.js_ can do the same thing as `executeScript()`?

Reply:
Yep! I'll push a change with the driverHelper function instead.